### ISSUE TITLE
[Bug] Comment field is not writable to, if very first comment

### ISSUE THREAD

**Issue by Olf0** [mentaljam/harbour-storeman#38]
Thanks for all the bug-fixes and enhancements lately, but unfortunately the comment field is not writable to (i.e. keyboard does not come up, when tapping on the comment field, but the whole window bounces), if very first comment (i.e. there are no prior comments for this application).
Tested with Storeman 0.0.18-1 (on a Jolla 1 phone under SailfishOS 2.1.3.7).

**Comment by Olf0** [comment]
Sorry, Storeman 0.0.19-1 (from OpenRepos) still exhibits this bug, thus please reopen this issue.

But hey, Storeman has become really nice and fast, it has become fun to use and test it (especially knowing that it likely is the only OpenRepos client working under SFOS 2.1.4 onwards).

You may consider "daring" to bump the release number to e.g. 0.9.0, as that is what Storeman feels like (a late beta or an release candidate), in order to not be frightening low (at 0.0.XY, currently) and hence scaring people off.

**Comment by Olf0** [comment]
Sorry for the delay.
Just retested and had it again: It is impossible to enter the comment field (now?) and the whole window still just jumps up and down, when trying to.

So I tried to log out and back in to OpenRepos: Logging in fails, although I have tried multiple times with the correct credentials (NB: the OpenRepos web site works in a browser).

Unfortunately, that only points to this issue being intermingled with the "Login status at OpenRepos" one.

I will keep on testing every now and then, until I (hopefully) have something more concise to report.

Never mind, Storeman is well working and already invaluable in its current state.

**Comment by Olf0** [comment]
I decided to continue right away:

- Closed and reopened Storeman (i.e. no OS reboot): Now I am able to log in to OpenRepos, again.
- Tried another first comment: Still the same behaviour (i.e. cannot enter the comment field, consequently the keyboard does not come up, but the window "jumps").
- In contrast to that, writing a comment works fine, when there are already existing comments.

So this issue is still reproducible for me.

**Comment by Olf0** [comment]
@mentaljam, I just ran into this issue again, with Storeman 0.0.28 under SFOS 2.1.4 on a Jolla 1 phone.  I retested (and successfully reproduced it, unfortunately) on another Jolla 1 phone and an Xperia X under SFOS 2.2.0!

It is reproducible for me across:
- Storeman restarts
- Logging out and back in to OpenRepos within Storeman 
- Reboots 

IOW, it always has been reproducible for me (since Storeman 0.0.18, with which I first noticed it), no matter what I try.

As you state, it is not reproducible for you: Is there any kind of logging, which may help understanding why I experience this issue, while you don't?  E.g., will grepping the journal (with *journalctl*) for Storeman create an output, which may be helpful for you? Maybe a specific RegEx comes to you mind, which is better suited than the simple string *Storeman*.

P.S.: So I will use the well-known workaround again: Write a first comment on OpenRepos in a Web-browser.  I.e., this issue is not at all tragic but really inconvenient.

**Comment by mentaljam** [comment]
@Pohli, yes there is an issue about incorrect auth status. But as you said there were no problems with entering a very first comment (no window bounces).

I leave this issue as unconfirmed until someone else point to it.

**Comment by Olf0** [comment]
Help is coming! ;)
Supposedly you can easily reproduce this issue by holding your device in landscape orientation.  It is sufficient to turn the device into landscape orientation after you have entered the comment section (of a yet uncommented app), but before placing the cursor in the comment field ("opening a comment for writing").  After placing the cursor in the comment field one can turn the device into landscape orientation and everything works as expected.
I still wonder, why this only occurs when it is the very first comment for an app on OpenRepos.

**Comment by Pohli** [comment]
Confirmed! It even occurs when entering the comment section in landscape mode.

**Comment by Olf0** [comment]
> Confirmed! It even occurs when entering the comment section in landscape mode.

Sure, I am always using the device (and hence Storeman, too) in landscape orientation.
My issue for half a year was to determine, when it does **not** happen (as you could not reproduce this issue).

**Comment by mentaljam** [comment]
I've just fixed it. The issue was in overlapping flickables. It was a long story and I happy that you, @Olf0, was dogged (do I use this word correctly?) enough to make me dig the real cause of it!

**Comment by Olf0** [comment]
I have never heard "dogged" in this context, but I am not a native English speaker, either.

"Proper language" words, which might express what you mean might be "insisting" (almost neutral, focusing on the topic) or "pertinent" (negative, focusing on the social behavior).

Wiktionary rsp. Aard2 (an Android app) with a couple of offline dictionaries and Wiktionary extracts installed (and other Wikipedia extracts, if you like) is very helpful for finding appropriate words.

P.S.: Wow, your turnaround time from debugging and fixing on your local machine, over submitting code to Github and building Storeman to publishing the new version on OpenRepos is extremely quick. :)

**Comment by Olf0** [comment]
Please note the edit above.
